apps/nextcloud/30.0.2/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nextcloud:
-    image: linuxserver/nextcloud:30.0.2
+    image: linuxserver/nextcloud:30.0.3
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There doesn't appear to be any significant difference between the two versions of this code snippet. Both are using a Nextcloud Docker container with specific parameters such as version, name format (${CONTAINER_NAME}), and network configuration.

If you have questions related to these specifications or other programming aspects, feel free to ask!